src/test/java/pathfinding/ConcreteGraphTest.java

```diff
@@ -6,12 +6,10 @@
 
 import static org.junit.Assert.*;
 
-public class ConcreteGraphTest
-{
+public class ConcreteGraphTest {
 
 	@Test
-	public void findPath() throws Exception
-	{
+	public void findPath() throws Exception {
```

Comment:
...why does this throw an exception? I'll allow this one too, it was in the original file, but I really need to fix this.